crates/sui-graphql-rpc/src/types/stake_subsidy.rs

```diff
@@ -4,7 +4,7 @@
 use super::big_int::BigInt;
 use async_graphql::*;
 
-/// Parameters related to subsiding staking rewards
+/// Parameters related to subsiding staking rewards.
```

Comment:
tbh not sure if this is rly needed

Reply:
I think I made this comment on some earlier PR, but "subsiding" is the wrong collocation here (e.g. "subsiding" does not go with "subsidy").

> Parameters that control the distribution of the stake subsidy.

Reply:
Thanks for catching this @amnn.